File: cosmos/Cosmos-API/src/main/java/com/planbtech/cosmos/model/Interfaces/IProfileConf.java
package com.planbtech.cosmos.model.Interfaces;

public interface IProfileConf {

    //get e set for id, woring and Homeoffice
    public void setId(Long id);
    public Long getId();

    public void setWorking(boolean working);
    public boolean getWorking();

    public void setHomeoffice(boolean homeoffice);
    public boolean getHomeoffice();

    public void setSalaryExpectation(float salaryExpectation);
    public float getSalaryExpectation();

}

File: cosmos/ProConnect/src/app/interfaces/iprofileconf.ts
export interface IProfileconf {
  id: number;
  working: boolean;
  homeoffice: boolean;
  salaryexpectation: number;


  getId(): number;
  getWorking(): boolean;
  getHomeoffice(): boolean;
  getSalaryexpectation(): number;

  setId(id: number): void;
  setWorking(working: boolean): void;
  setHomeoffice(homeoffice: boolean): void;
  setSalaryexpectation(salaryexpectation: number): void;
}
